### PR TITLE
[zh-cn] Replace deprecated fullversion shortcode

### PR DESCRIPTION
**Pull request by rolfedh**
Fixes https://github.com/kubernetes/website/issues/41484 - [zh-cn] Replace deprecated fullversion shortcode #41484
/area localization
cc: @seokho-son
Related to https://github.com/kubernetes/website/issues/41005

### CODE REVIEW

File: content/zh-cn/docs/tasks/tools/install-kubectl-windows.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ The following methods exist for installing kubectl on Windows:
 ### 用 curl 在 Windows 上安装 kubectl {#install-kubectl-binary-with-curl-on-windows}
 
 <!--
-1. Download the [latest release {{< param "fullversion" >}}](https://dl.k8s.io/release/{{< param "fullversion" >}}/bin/windows/amd64/kubectl.exe).
+1. Download the [latest release {{< skew currentPatchVersion >}}](https://dl.k8s.io/release/v{{< skew currentPatchVersion >}}/bin/windows/amd64/kubectl.exe).
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/tools/install-kubectl-windows.md:58]: Line 57-59 can be removed, which is a dup of line 60-63.

Comment (rolfedh) [reply content/zh-cn/docs/tasks/tools/install-kubectl-windows.md:58]: Thanks, @windsonsea. That change is outside the scope of this PR which is to simply replace the obsolete attribute wherever it appears. Could you create a separate issue/PR to handle the removal?

Comment (windsonsea) [reply content/zh-cn/docs/tasks/tools/install-kubectl-windows.md:58]: Ok, marked and will do.

**Pull request merged**